core/src/com/po/fuck/view/classdrawers/factories/EntityDrawerFactory.java

```diff
@@ -0,0 +1,42 @@
+package com.po.fuck.view.classdrawers.factories;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.math.Vector2;
+import com.po.fuck.model.Constants;
+import com.po.fuck.model.Entity;
+import com.po.fuck.model.Player;
+import com.po.fuck.model.enemies.BasicEnemy;
+import com.po.fuck.view.CenterDrawer;
+import com.po.fuck.view.HealthBar;
+import com.po.fuck.view.Renderer;
+import com.po.fuck.view.classdrawers.ClassDrawer;
+
+public class EntityDrawerFactory {
+    
+    static {
+        Renderer.addDrawer(Player.class, EntityDrawerFactory.get(Player.class, new Sprite(new Texture("FUCKerWithoutHands2.png"))));
+        Renderer.addDrawer(BasicEnemy.class, EntityDrawerFactory.get(BasicEnemy.class, new Sprite(new Texture("player2.png"))));
+    }
+
+    public static <T extends Entity> ClassDrawer<T> get(Class<T> clz, Sprite other_sprite){
```

Comment:
`other_sprite` - not java style naming, also not very obvious, maybe add javadoc or rename it